### PR TITLE
Fix parsing/printing double precision

### PR DESCRIPTION
**Pull request by notmgsk**
See #12

### CODE REVIEW

File: src/expression.rs [hunk 1]
```diff
@@ -17,12 +17,19 @@ use crate::{imag, instruction::MemoryReference, real};
...
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq, Error)]
 pub enum EvaluationError {
     Incomplete,
 }
 
+impl fmt::Display for EvaluationError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "SuperError is here!")
```
Comment (kalzoo) [inline src/expression.rs:29]: this looks unfinished

File: Cargo.toml [hunk 2]
```diff
@@ -12,3 +12,4 @@ nom = "6.1.0"
 num-complex = "0.3.0"
 petgraph = "0.5.1"
 serde = { version = "1.0.125", features = ["derive"] }
+thiserror = "1.0.26"
```
Comment (kalzoo) [inline Cargo.toml:15]: Can you provide an MR description pitching this change?

**Review by kalzoo — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 2 above)
Comment (notmgsk) [reply Cargo.toml]: removed

File: src/expression.rs (hunk 1 above)
Comment (notmgsk) [reply src/expression.rs]: removed

**Comment by kalzoo** [comment]
Good call, let's remove `pretty_float!` altogether then

**Review by genos — APPROVED** [review]

**Pull request merged**